qube/create/templates/cli/cli_java/{{ cookiecutter.project_slug }}/.dependabot/config.yml

```diff
@@ -0,0 +1,9 @@
+version: 1
```

Comment:
Why are you now using the old v1 version again and deleted the new v2 version (.github/dependabot.yml)

Reply:
This is how it was in qOffer 2.0. You suggested me to use the file as in this project

Reply:
Then I must have been wrong with it being correct in qOffer 2.0 or we had a misunderstanding. Sorry for that.

https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates

The file must be stored directly under the .github folder and it should be version 2 and not version 1.

Sorry!

Reply:
https://github.com/qbicsoftware/qoffer-2-portlet/blob/development/.github/dependabot.yml

This is what I mean.
Maybe you were looking at the wrong branch or so.

Reply:
addressed in 712bc2f

Reply:
Dependabot may notify you that the file is supposed to be in a .github folder.

`You must store this file in the .github directory of your repository. `
https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates

But it should still work as long as they support the old paths and versions.

Reply:
Ahh, @jenniferboedker addressed this already.

Well then :rocket: